DataConnectors/GoogleWorkspaceReports/GWorkspaceReportsAPISentinelConnector/__init__.py

```diff
@@ -14,18 +14,40 @@
 import os
 import time
 import re
+from .state_manager import StateManager
 
 customer_id = os.environ['WorkspaceID'] 
 shared_key = os.environ['WorkspaceKey']
 pickle_str = os.environ['GooglePickleString']
 pickle_string = base64.b64decode(pickle_str)
-SCOPES = ['https://www.googleapis.com/auth/admin.reports.audit.readonly']
-activities = ["login", "calendar", "drive", "admin", "mobile", "token", "user_accounts"]
+connection_string = os.environ['AzureWebJobsStorage']
 logAnalyticsUri = os.environ.get('logAnalyticsUri')
+SCOPES = ['https://www.googleapis.com/auth/admin.reports.audit.readonly']
+activities = [
+            "access_transparency", 
+            "admin",
+            "calendar",
+            "chat",
+            "drive",
+            "gcp",
+            "gplus",

```

Comment:
While testing getting ModuleNotFoundError. Please fix the issue.

![image](https://user-images.githubusercontent.com/78137909/120622292-4bb3de80-c47c-11eb-9ca4-0ce1fcb9f550.png)

Reply:
You might be used old version of arm template azuredeploy_Connector_GWorkspaceReportsAPI_AzureFunction.json. Pls use the last version for deploying.